autoload/doom.wad/brghtmps.lmp

```diff
@@ -0,0 +1 @@
+TEXTURE SW2STON2 REDONLY
```

Comment:
I'd prefer if we could have an *optional* fourth parameter on each line to indicate if a beightmap applies to Doom 1 or 2 (no such parameter would imply "both"). This way we would (1) keep our format compatible to Doom Retro's (which is the purpose of this change, c.f. PR title) and (2) have less external files to rely on which in turn rely on the IWAD file names themselves (remember my concerns regarding bfgdoom.wad, udoom.wad, freedoom1.wad, d2.wad, etc.).

Reply:
Alternative approach: How about we support autoload directories based on the internal gamemission name (i.e. doom, doom2, pack_tnt, pack_plut, etc.)? Then we'd end up with only one pack_rekkr directory for example and would be independent of the actual IWAD file name - because game mission is detected based on content if in doubt (c.f. `CheckIWAD()`).

Reply:
At this point, we can make a zip/wad resource file with all the optional content and identify everything by hash. But I like the current system, I think the cases of udoom.wad d2.wad are very rare.

Reply:
Is there anything that speaks against supporting the fourth field in the BRGHTMPS lump lines to differentiate between Doom 1 and 2? We could have a single brghtmps.lmp in autoload/doom-all and let the parser do the rest.

Reply:
Okay, we can get rid of the `brghtmps.lmp` files in the `doom.wad` and `doom2.wad` directories, but we'll keep the rest of the files, right?

Reply:
In general, yes. Though, we could also get rid of the file in the tnt.wad directory and merge it into the one in doom-all. The textures which get assigned a brightmap in this file do not even exist in any other IWAD.